website/docs/documentationFile.md

```diff
@@ -0,0 +1,43 @@
+---
+title: documentation file
+---
+
+Enables C# xmlDocumentationFile
+
+# Usage (2) #
+The Documentation File is used for adding the [xml comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/) added to functions/ variables, to a dll that has been packed inside a framework or other C# related project.
+This can then be referenced inside another project by placing it next to the corresponding dll.
+
+This feature sets the [Documentation File](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/output#documentationfile) option inside csproj for each corresponding [configuration](https://premake.github.io/docs/configurations/)
+
+## 1) Default ##
+
+When you put **documentationfile** inside the project configuration, the following filename/path will be generated:
+```%{targetdir}/%{prj.name}.xml```
+```lua
+documentationfile ""
+```
+
+## 2) Custom Directory ##
+
+When you put the following inside the project configuration the following filename/path will be generated:
+```bin\test\%{prj.name}.xml```
+
+```lua
+documentationfile "bin/test"
+```
+<b>The path is relative to the project [location](https://premake.github.io/docs/location/)
```

Comment:
I think that all other paths as written relative from premake5.lua

There are indeed (pre/post build/link) commands which requires extra `%[]` for that.

Wonder if it is relevant to not be consistent with that api...

Reply:
that is what i meant it is relevant to the project configuration (premake5.lua)

Reply:
`documentationfile "bin/test"` versus `documentationfile "%{prj.location}/bin/test"`
Other APIs use the second version.

Reply:
changed it!